src/Specular/FRP/Base.purs

```diff
@@ -120,6 +120,13 @@ type Unsubscribe = Effect Unit
 -- | events coincide), but it's not very useful.
 newtype Event a = Event (Node a)
 
+-- append biased towards first argument - if appended events occur simultaneously then the first one is returned
```

Comment:
Maybe add a proper docstring? The behavior is not that obvious (`Monoid a => Monoid (Event a)` would also be a sensible choice)

Reply:
Sure, will do.

`Monoid a => Monoid (Event a)` - you mean appending simultaneous events of type `Monoid a => a`? That would not have lawful `mempty`, or we have anything like `always :: Event a`?

Reply:
Sorry, I meant `Semigroup a => Semigroup (Event a)`.

But then `mempty = never` would also work fine. `never` is never simultaneous with other event, so the event value would stay unchanged, therefore `never <> e = e`.

Reply:
> would also be a sensible choice

To be clear, I think `leftmost` is better. I rarely care about simultaneous Events, and this would unnecessarily impose a Semigroup constraint on the content. But would be perhaps more "Haskell-y"